lib/html_scrubber_util.rb

```diff
@@ -10,7 +10,11 @@ def sanitize_html(value)
     ActionController::Base.helpers.sanitize(
       value,
       tags: Loofah::HTML5::SafeList::ACCEPTABLE_ELEMENTS.dup.delete("select").add("style"),
-      attributes: Loofah::HTML5::SafeList::ACCEPTABLE_ATTRIBUTES.dup.add("style")
+      attributes: Loofah::HTML5::SafeList::ACCEPTABLE_ATTRIBUTES.dup
+        .add("style")
+        .add("data-toggle")
+        .add("data-slide-to")
+        .add("data-target")
```

Comment:
Small request, could we change this to use `merge` similar to the update to pdf_scrubber_util? https://github.com/ideacrew/enroll/blob/d1febb2bc9a00a73862701605bf5dc95a67f8acc/lib/pdf_scrubber_util.rb#L12

Reply:
Yessir, file is prettified!